File: Java/CatchPhrase/src/CatchPhrase.java
import java.util.Scanner;
public class CatchPhrase {

	public static void main(String[] args) 
	{
		Scanner input = new Scanner (System.in);
		String userInput;
		
		System.out.println ("List of Characters: The Donald, The Soup Nazi, and Julia Child");
		System.out.println ("Type a character name:");
		userInput = input.nextLine();
		
		if (userInput.equalsIgnoreCase ("The Donald"))
		{
			System.out.println ("You're Fired!");
		}
		
		if (userInput.equalsIgnoreCase ("The Soup Nazi"))
		{
			System.out.println ("No Soup for You!");
		}
		
		if (userInput.equalsIgnoreCase ("Julia Child"))
		{
			System.out.println ("Needs more butter!!!");
		}

	}

}

File: Catchphrase/Catchphrase/Catchphrase.cpp
#include <iostream>
#include <string>
using namespace std;

int main()
{
	string userInput;
	
	cout << ("List of characters: The Donald, The Soup Nazi, Julia Child") << endl;
	cout << ("Enter a character: ") << endl;
	getline (cin,userInput);	//Allows string comparison to ignore white space

	if (userInput == "The Donald")
	{
		cout << ("You're Fired!") << endl;
	}
	
	if (userInput == "The Soup Nazi")
	{
		cout << ("No Soup for You!") << endl;
	}
	
	if (userInput == "Julia Child")
	{
		cout << ("Needs more butter!!!") << endl;
	}
}